portfolio/src/main/webapp/style.css

```diff
@@ -75,28 +76,101 @@ ul#contact li {
   padding: 20px;
 }
 
-#gallery {
-  margin: 5px;
-  border: 1px solid #ccc;
-  float: left;
-  width: 180px;
-  padding: 6px;
-  background-color:rgb(132, 202, 243)
+.wrapperGallery {
+    margin: 0 20vw;
+
 }
 
-#gallery:hover {
-  border: 5px solid #777;
+#photoGallery {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    gap: 10px;
+    grid-template-areas:
+    "h1 h1 h1"
+    "img1 img2 img3"
+    "img4 img5 img6"
+    "img7 img8 img9";
 }
 
-#gallery img {
-  width: 100%;
-  height: auto;
-  display: inline-block;
-  margin-left: auto;
-  margin-right: auto;
+#photoGallery .galleryImg {
+    width: 100%;
+    height: 160px;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    cursor: pointer;
+}
+
+#photoGallery h1 {
+    grid-area: h1;
+    text-align: center;
+}
+
+#photoGallery .img1 {
```

Comment:
couple of things that I won't make you change since it's a new week, but for the sake of future work...
1. it's better to give your files a more descriptive name so you don't have to open up the image file every time you want to remember what it is.
2. while this pattern with css is fairly common and looks very pretty it's not particularly accessible, screenreaders and such can't tell there's a picture there, it's better to try and do this sort of thing with an image tag.